public/docs/ts/latest/guide/dependency-injection.jade

```diff
@@ -84,7 +82,7 @@ include ../../../../_includes/_util-fns
 :marked
   Now we create a car by passing the engine and tires to the constructor.
 
-+makeExample('dependency-injection/ts/app/car/car-creations.ts', 'car-ctor-instantiation', 'Simple Car')(format=".")
++makeExample('dependency-injection/ts/app/car/car-creations.ts', 'car-ctor-instantiation', '')(format=".")
```

Comment:
Why remove the title 'Simple Car'?  Doing so makes it difficult to relate this code to the derived "Super Car" coming up soon.